posthog/test/test_client.py

```diff
@@ -112,7 +112,7 @@ def test_get_active_feature_flags(self, patch_decide):
         }
 
         client = Client(FAKE_TEST_API_KEY, on_error=self.set_fail, personal_api_key=FAKE_TEST_API_KEY)
-        variants = client._get_active_feature_variants("some_id", None, None, None)
+        variants = client._get_active_feature_variants("some_id", None, None, None, True)
```

Comment:
Here, assert decide was called with the 'right' geoip_disable value.

Also add a similar test for capture, where you're not manually calling `client._get_active_feature_variants` but let capture call it